src/components/ToDoComponent/TodoComponent.tsx

```diff
@@ -58,31 +68,38 @@ const TodoComponent = () => {
           </Text>
         </TouchableOpacity>
       </View>
-      <View style={{ paddingVertical: 35 }}>
-        {tasks?.length === 0 ? (
-          <Text style={TodoStyles.taskNotFound}>No tasks found</Text>
-        ) : (
-          tasks
-            ?.map((task) => {
-              const { title, assigned_by } = task?.attributes;
-              return (
-                <Card
-                  posStyle={tasks.indexOf(task) !== 0 ? 'absolute' : 'relative'}
-                  key={task.id}
-                  item={task}
-                  changecard={changeCardFunction}
-                  removeCard={removeCard}
-                  disabled={disabled}
-                  setDisabled={setDisabled}
-                  title={title}
-                  assigned_by={assigned_by}
-                />
-              );
-            })
-            .reverse()
-        )}
-        <View style={TodoStyles.shodowcard} />
-      </View>
+      {loader ? (
+        <ActivityIndicator size="large" />
+      ) : (
+        <View style={{ paddingVertical: 35 }}>
+          {tasks?.length === 0 ? (
+            <Text style={TodoStyles.taskNotFound}>No tasks found</Text>
+          ) : (
+            tasks
+              ?.map((task) => {
+                const { title, assigned_by } = task?.attributes;
+                console.log(assigned_by);
+                return (
+                  <Card
+                    posStyle={
+                      tasks.indexOf(task) !== 0 ? 'absolute' : 'relative'
+                    }
+                    key={task.id}
+                    item={task}
+                    changecard={changeCardFunction}
+                    removeCard={removeCard}
+                    disabled={disabled}
+                    setDisabled={setDisabled}
+                    title={title}
+                    assigned_by={assigned_by === '' ? 'Unknown' : assigned_by}
```

Comment:
you may also write 
`assigned_by ??  'unknown'`
please read about `coalescing operator`
https://javascript.info/nullish-coalescing-operator

Reply:
Will change in the ui fix